eng/common/pipelines/templates/steps/git-push-changes.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  PRBranchName: not-specified
+  PROwner: azure-sdk
+  CommitMsg: not-specified
+  RepoOwner: Azure
+  RepoName: $(Build.Repository.Name)
+  PushArgs:
+  WorkingDirectory: $(System.DefaultWorkingDirectory)'
+  ScriptDirectory: eng/common/scripts
+  SkipCheckingForChanges: false
+  RemoteName: ''
```

Comment:
Lets completely remove this as a parameter as I don't think we need it and it will most likely cause more issues then it solves.

Reply:
I am afraid `create-pull-request` has no way to create the fork if completely remove it

Reply:
It doesn't actually create the fork it only creates the remote name. The fork or repo you are pushing to is controlled by the GitUrl that is created,

Reply:
Those are controlled by the RepoName and PROwner.